jupyter/d2l-java/chapter_multilayer-perceptrons/dropout.ipynb

```diff
@@ -0,0 +1,812 @@
+    "            }\n",
+    "\n",
+    "            testAccuracy[epoch-1] = accuracyVal/testIter.size();\n",
+    "            epochCount[epoch-1] = epoch;\n",
+    "            accuracyVal = 0f;\n",
+    "            System.out.println(\"Finished epoch \" + epoch);\n",
+    "}\n",
+    "\n",
+    "System.out.println(\"Finished training!\");"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "String[] lossLabel = new String[trainLoss.length + testAccuracy.length + trainAccuracy.length];\n",
+    "\n",
+    "Arrays.fill(lossLabel, 0, trainLoss.length, \"train loss\");\n",
+    "Arrays.fill(lossLabel, trainAccuracy.length, trainLoss.length + trainAccuracy.length, \"train acc\");\n",
+    "Arrays.fill(lossLabel, trainLoss.length + trainAccuracy.length,\n",
+    "                trainLoss.length + testAccuracy.length + trainAccuracy.length, \"test acc\");\n",
+    "\n",
+    "Table data = Table.create(\"Data\").addColumns(\n",
+    "            FloatColumn.create(\"epochCount\", ArrayUtils.addAll(epochCount, ArrayUtils.addAll(epochCount, epochCount))),\n",
+    "            FloatColumn.create(\"loss\", ArrayUtils.addAll(trainLoss, ArrayUtils.addAll(trainAccuracy, testAccuracy))),\n",
+    "            StringColumn.create(\"lossLabel\", lossLabel)\n",
+    ");\n",
+    "\n",
+    "render(LinePlot.create(\"\", data, \"epochCount\", \"loss\", \"lossLabel\"),\"text/html\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concise Implementation\n",
+    "\n",
+    "Using DJL, all we need to do is add a `Dropout` layer\n",
+    "after each fully-connected layer, \n",
+    "passing in the dropout probability\n",
+    "as the only argument to its constructor.\n",
+    "During training, the `Dropout` layer will randomly\n",
+    "drop out outputs of the previous layer\n",
+    "(or equivalently, the inputs to the subsequent layer)\n",
+    "according to the specified dropout probability.\n",
+    "When the model is not in training mode,\n",
+    "the `Dropout` layer simply passes the data through during testing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "attributes": {
+     "classes": [],
+     "id": "",
+     "n": "6"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "SequentialBlock net = new SequentialBlock();\n",
+    "net.add(Blocks.batchFlattenBlock(784));\n",
+    "net.add(Linear.builder().setOutChannels(256).build());\n",
+    "net.add(Activation::relu);\n",
+    "net.add(Dropout.builder().optProbability(dropout1).build());\n",
+    "net.add(Linear.builder().setOutChannels(256).build());\n",
+    "net.add(Activation::relu);\n",
+    "net.add(Dropout.builder().optProbability(dropout2).build());\n",
+    "net.add(Linear.builder().setOutChannels(10).build());\n",
+    "net.setInitializer(new NormalInitializer());"
```

Comment:
Make sigma to 0.01 to follow the book

Reply:
NormalInitializer comes with a 0.01f sigma as default. Should I still give the value even then ?